systems/subscriber/sim-pool/pkg/db/model.go

```diff
@@ -0,0 +1,17 @@
+package db
+
+import (
+	"gorm.io/gorm"
+)
+
+type Sim struct {
+	gorm.Model
+	Iccid          string `gorm:"index:idx_iccid,unique"`
+	Msisdn         string
+	Is_allocated   bool
```

Comment:
CamelCase naming if need to explictly define column name use gorm:"column:is_allocated"